packages/devtools/lib/src/ui/elements.dart

```diff
@@ -410,6 +414,28 @@ class CloseButton extends CoreElement {
   CloseButton() : super('div', classes: 'close-button');
 }
 
+class TextField extends CoreElement {
+  TextField({String c}) : super('input', classes: c) {
+    setAttribute('type', 'text');
+  }
+
+  String get value => (this.element as InputElement).value;
+
+  void set value(String value) {
+    (this.element as InputElement).value = value;
+  }
+
+  /// Subscribe to the [onKeyUp] event stream with a no-arg handler.
+  StreamSubscription<Event> changed(void handle(String value)) {
+    return onKeyUp.listen((KeyboardEvent e) {
+      e.stopImmediatePropagation();
+      if (e.target == element) {
```

Comment:
when will the target be something other than element?

Reply:
Shouldn't be made it an assert.